### PR TITLE
Minor additions to components - mostly links

### PR DESCRIPTION
**Pull request by sarah11918**
Mostly adding lists to things that now exist (integrations, directives page) and an example of `set:html`

### CODE REVIEW

**Comment by delucis** [comment]
This all LGTM! I do wonder if that code block you added the directive example to is getting a little unwieldy? Maybe we need to think about breaking it up at some point.

File: src/pages/en/core-concepts/astro-components.md [hunk 1]
```diff
@@ -100,6 +100,10 @@ const myFavoritePokemon = [/* ... */];
   {myFavoritePokemon.map((data) => <li>{data.name}</li>)}
 <ul>
 
+<!--Use a template directive to inject an unescaped HTML string into an element: -->
```
Comment (sarah11918) [inline src/pages/en/core-concepts/astro-components.md:103]:
```suggestion
<!-- Use a template directive to inject an unescaped HTML string into an element: -->
```

**Comment by sarah11918** [comment]
> This all LGTM! I do wonder if that code block you added the directive example to is getting a little unwieldy? Maybe we need to think about breaking it up at some point.

I'm OK to think about that at some point. I don't think it's blocking for now?  :)

**Comment by delucis** [comment]
> I don't think it's blocking for now? :)

Agreed!

**Pull request merged**